src/runtime_src/core/pcie/driver/linux/xocl/xocl_xclbin.c

```diff
@@ -153,7 +153,7 @@ static int xgq_xclbin_post_download(xdev_handle_t xdev, void *args)
 		/* after download, update clock freq */
 		topo = (struct clock_freq_topology *)
 		    (((char *)(arg->xclbin)) + hdr->m_sectionOffset);
-		ret = xocl_xgq_freq_scaling_by_topo(xdev, topo, 0);
+		ret = xocl_xgq_freq_scaling_by_topo(xdev, topo, 1);
```

Comment:
since we changed the behavior on both vck5000 and v70. we'd better test it in case current valid might fail.